docs/_static/theme_overrides.css

```
@@ -0,0 +1,13 @@
+/* override table width restrictions */
+@media screen and (min-width: 767px) {
+
+   .wy-table-responsive table td {
+      /* !important prevents the common CSS stylesheets from overriding
+         this as on RTD they are loaded after this stylesheet */
+      white-space: normal !important;
+   }
+
```

Optional: Tables are transparent. When the table grows bigger, the background changes when it passes the "page" and become gray (hard to read).

```suggestion

  .wy-table-responsive table {
      background-color: white;
   }
```